packages/react/src/profile/useProfile.ts

```diff
@@ -47,38 +56,47 @@ export function useProfile({
               observerId,
             }),
           ),
+          skip,
         }),
       ),
     ),
   );
 
+  if (skip) {
+    return {
+      data: undefined,
+      error: undefined,
+      loading: false,
+    } as SkippableReadResult<A, Profile, NotFoundError | UnspecifiedError>;
```

Comment:
I don't like the need for `as ReturnType` but I didn't figure out how to do it better. Any ideas?